californium-core/src/main/java/org/eclipse/californium/core/coap/Request.java

```diff
@@ -240,6 +240,9 @@
 	/** Contextual information about this request */
 	private Map<String, String> userContext;
 
+	/** Indicates if handling the response caused an error. */
```

Comment:
Please add since 2.3

Reply:
just to be sure you want to add it on private attribute too ?

Reply:
I used to add it everywhere. It may help to understand the implementation and differences between the versions.

Reply:
OK I add it